doc/radosgw/multisite.rst

```diff
@@ -5,7 +5,7 @@ RGW Multisite
 .. versionadded:: Jewel
 
 From Ceph release Jewel and beyond, you may configure each :term:`Ceph Object
-Gateway` to work in an active active zone configuration, allowing for writing to
```

Comment:
active active means both zones can write data, in old way only master zone can write data.